V2_MIGRATION_GUIDE.md

```diff
@@ -97,6 +97,20 @@ Use `createUser(email:username:password:connection:userMetadata:rootAttributes:`
 
 Use `userInfo(withAccessToken:)` instead.
 
+#### `tokenExchange(withParameters:)`
+
+Use `tokenExchange(withCode:codeVerifier:redirectURI:)` instead. To pass custom parameters, use the `parameters(_:)` method from `Request`:
```

Comment:
Similarly, this should really be called `codeExchange` - since you're doing Proof Key for *Code Exchange* (not sure what Android calls it)

Reply:
Good catch, renamed the method in https://github.com/auth0/Auth0.swift/pull/573/commits/8f5a6561e3739ce4f122668f5ef2135481789ce4.
Android calls it `token()`.